src/app/cube/user-profile/components/profile-header/profile-header.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ProfileService } from 'app/core/profiles.service';
+import { UserService } from 'app/core/user.service';
+import { BehaviorSubject } from 'rxjs';
+
+@Component({
+  selector: 'clark-profile-header',
+  templateUrl: './profile-header.component.html',
+  styleUrls: ['./profile-header.component.scss']
+})
+export class ProfileHeaderComponent implements OnInit {
+  private _user = new BehaviorSubject<any>({});
+  @Input() set user(user: any) {
+    this._user.next(user);
+  }
+  get user() {
+    return this._user.value;
+  }
```

Comment:
Why is user a behavior subject?

Reply:
It's being used as an observable

Reply:
Could we write a quick comment about how this is used to register a user change for a profile?  I.e. seeing user x's profile then loading user y's.  Without this the component won't update to show user y's profile